connectors/src/connectors/slack/bot.ts

```diff
@@ -9,7 +10,6 @@ import type {
   LightAgentConfigurationType,
   ModelId,
   Result,
-  UserMessageType,
 } from "@dust-tt/types";
```

Comment:
Ideally, we should not import any type from type, if possible.

Reply:
The end goal (talked with @flvndvd) would be to fold dust-tt/types into front and remove this shared "fourre tout" that is dust-tt/types